scss/_navbar.scss

```diff
@@ -115,6 +115,12 @@
   @include hover-focus {
     text-decoration: none;
   }
+
+  &::before {
```

Comment:
A comment explaining what this is for (e.g. `// Hamburger menu icon`) would be good.